src/lib/blocks.js

```diff
@@ -110,6 +110,75 @@ export default function (vm) {
         return [[myself, '_myself_']].concat(spriteMenu());
     };
 
+    const variablePropertyMenu = function (thisValue) {
```

Comment:
In order to access the "of" block (and then its `OBJECT` input's value), I need to be able to access the "this" value of where this function is run. This argument should probably be named better than `thisValue`, though.